app/components/file_attachment_list/file_attachment.js

```diff
@@ -211,5 +233,18 @@ const getStyleSheet = makeStyleSheetFromTheme((theme) => {
             alignItems: 'center',
             justifyContent: 'center',
         },
+        moreImagesWrapper: {
+            ...StyleSheet.absoluteFill,
+            justifyContent: 'center',
+            alignItems: 'center',
+            backgroundColor: 'rgba(0,0,0,.6)',
```

Comment:
Let's keep the current recommended color, even on the darker themes it will display nicely.

**Mattermost Dark Theme** | **Windows Dark Theme**
------------ | -------------
![image](https://user-images.githubusercontent.com/48559658/68621008-b8334b00-049c-11ea-9408-431fb2ae2738.png) | ![image](https://user-images.githubusercontent.com/48559658/68621064-d5681980-049c-11ea-9b2a-d78688bb6318.png)